data/src/main/java/com/microsoft/azure/kusto/data/exceptions/KustoDataExceptionBase.java

```diff
@@ -1,10 +1,10 @@
 package com.microsoft.azure.kusto.data.exceptions;
 
-public abstract class KustoDataException extends Exception {
```

Comment:
why not abstract (the new name even suggests its abstract)

Reply:
I vaguely remember having a reason, but I'm not sure what I was thinking :-D

Reverting back to abstract.